kola/harness.go

```diff
@@ -367,11 +368,26 @@ func RunTests(patterns []string, channel, offering, pltfrm, outputDir string, ss
 
 	if !skipGetVersion {
 		plog.Info("Creating cluster to check semver...")
+
 		version, err := getClusterSemver(flight, outputDir)
 		if err != nil {
 			plog.Fatal(err)
 		}
 
+		// If the version is > 3033, we can safely use user-data instead of custom-data for
+		// provisioning the instance on Azure.
+		if version.Compare(semver.Version{Major: 3034}) == 1 && pltfrm == "azure" {
```

Comment:
```suggestion
		if version.GreaterThanOrEqual(semver.Version{Major: 3034}) && pltfrm == "azure" {
```

Reply:
Where did you find it? There is no mention of this in the godoc (https://pkg.go.dev/github.com/coreos/go-semver/semver) neither in the vendors 🤔 
I only found `LessThan`, that's why I used `Compare`

Reply:
Ah, then it's the wrong go version library I looked up^^ Then how about `!LessThan`?